### PR TITLE
Make string in search template translatable. Re: #160

### PR DESCRIPTION
**Pull request by bobmatyas**
**Description**

The only template file that needed to be updated was `search.html`. I added the translations there.

### CODE REVIEW

**Comment by luminuu** [comment]
@bobmatyas Just checked on this, you've added the translation function in a HTML file, which is not going to work. You'd need to move this to a pattern as PHP file and then call that pattern inside the `search.html`.

**Comment by bobmatyas** [comment]
@luminuu Shoot, I misunderstood how that what was done. I'll make a PHP pattern for it.

File: patterns/search.php [hunk 1]
```diff
@@ -0,0 +1,15 @@
+<?php
+/**
+ * Title: Search
+ * Slug: twentytwentyfive/search
+ * Categories: search
+ * Description: Translatable search form
```
Comment (juanfra) [inline patterns/search.php]: This one needs to be hidden. We need to rename the file to `hidden-search` and change the docblocks. For hidden patterns we can avoid the description.

```suggestion
 * Title: Search
 * Slug: twentytwentyfive/hidden-search
 * Inserter: no
```

File: templates/search.html [hunk 2]
```diff
@@ -3,7 +3,7 @@
 <!-- wp:group {"tagName":"main","style":{"spacing":{"margin":{"top":"var:preset|spacing|60"}}},"layout":{"type":"constrained"}} -->
 <main class="wp-block-group" style="margin-top:var(--wp--preset--spacing--60)">
 	<!-- wp:query-title {"type":"search"} /-->
-	<!-- wp:search {"label":"Search","showLabel":false,"buttonText":"Search","style":{"border":{"radius":"50px"}},"fontSize":"medium"} /-->
+	<!-- wp:pattern {"slug":"twentytwentyfive/search"} /-->
```
Comment (juanfra) [inline templates/search.html]: We'll need to update this one once we change the filename.

```suggestion
	<!-- wp:pattern {"slug":"twentytwentyfive/hidden-search"} /-->
```

**Review by juanfra — CHANGES_REQUESTED** [review]
Thank you for working on this @bobmatyas - I left a few comments we need to address before merging.

File: patterns/search.php (hunk 1 above)
Comment (bobmatyas) [reply patterns/search.php]: @juanfra this change has been made. I didn't know about the hidden pattern syntax. Thanks for explaining :)

File: templates/search.html (hunk 2 above)
Comment (bobmatyas) [reply templates/search.html]: @juanfra this change has been made. I didn't know about the hidden pattern syntax. Thanks for explaining :)

File: patterns/hidden-search.php [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+ * Inserter: no
+ *
+ * @package WordPress
+ * @subpackage Twenty_Twenty_Five
+ * @since Twenty Twenty-Five 1.0
+ */
+
+?>
+
+<!-- wp:search {"label":"<?php esc_html_e( 'Search', 'twentytwentyfive' ); ?>","showLabel":false,"buttonText":<?php echo esc_attr_x( 'Search', 'button label', 'twentytwentyfive' ); ?>","style":{"border":{"radius":"50px"}},"fontSize":"medium"} /-->
```
Comment (juanfra) [inline patterns/hidden-search.php]: There's a missing quote at the start of the `ButtonText` value.

**Review by juanfra — COMMENTED** [review]
Thank you @bobmatyas - There's one last thing I found and this one should be good to merge.

File: patterns/hidden-search.php (hunk 3 above)
Comment (juanfra) [inline patterns/hidden-search.php]: Let's fix the `buttonText`. And we could also set the `borderColor` to match the design, now that we're touching these files. Thanks, Bob!

```suggestion
<!-- wp:search {"label":"<?php esc_html_e( 'Search', 'twentytwentyfive' ); ?>","showLabel":false,"buttonText":"<?php echo esc_attr_x( 'Search', 'button label', 'twentytwentyfive' ); ?>","style":{"border":{"radius":"50px"}},"fontSize":"medium","borderColor":"opacity-20"} /-->
```

Comment (bobmatyas) [reply patterns/hidden-search.php]: Fixed below:

**Comment by bobmatyas** [comment]
@juanfra Changes made in https://github.com/WordPress/twentytwentyfive/pull/357/commits/0c12404c8190ef69352acb2403bf3ba9ee668517

**Review by juanfra — APPROVED** [review]
Thank you @bobmatyas!

**Pull request merged**